### DISCUSSION TITLE
Re-enable former missing elements (globally)

### DISCUSSION THREAD

**Discussion by Joorren** [engram-design/FeedMe#1025] [Ideas]
### Description
When having the setting 'Disable missing elements globally' enabled, entries that are not present in the import will get disabled globally. 

It would be helpful if this setting would also exist to re-enable deactivated entries when they are present again in the next import.

**Comment by daltonrooney** [comment]
@Joorren This is the way the plugin should work, assuming you've set a default status of "enabled" for your elements. See @angrybrad's comment [here](https://github.com/craftcms/feed-me/issues/666#issuecomment-742835566)

If you've set the default status and your elements _still_ aren't enabled after they've been returned to the feed, that would be a bug, not an enhancement.

**Comment by Joorren** [comment]
@daltonrooney it seems to only enable the entry for the site the feed is bound to. The other languages of these imports just stay disabled...
